xonsh/xontribs.json

```diff
@@ -170,71 +170,71 @@
    "url": "http://exofrills.org",
    "install": {
     "conda": "conda install -c conda-forge xo",
-    "pip": "pip install exofrills"}
+    "pip": "xip install exofrills"}
```

Comment:
Pretty much yes, because `pip` generally refers to the system (or venv) pip, whereas `xip` refers to the pip matching xonsh.

Running `pip install xontrib-spam` will only put it in the right place if xonsh is installed in the system environment. Which is not the case for conda and homebrew.

The only entry that you might not want to do this is the installation for `xonsh` itself, and frankly I'm not even sure why that's in this file.